llvm/lib/Transforms/Scalar/LowerExpectIntrinsic.cpp

```diff
@@ -369,9 +369,21 @@ static bool lowerExpectIntrinsic(Function &F) {
     if (BranchInst *BI = dyn_cast<BranchInst>(BB.getTerminator())) {
       if (handleBranchExpect(*BI))
         ExpectIntrinsicsHandled++;
+      else {
+        SmallVector<uint32_t> Weights;
+        if (extractBranchWeights(*BI, Weights))
+          misexpect::checkMissingAnnotations(*BI, Weights,
+                                             /*IsFrontendInstr=*/false);
```

Comment:
Confused about how this relates to enabling checking for frontend instrumentation since all of these set that flag to false?

Reply:
oh no! You're right, this is basically dead. Let me go through this and try to figure out why I thought this was working as intended.